### PR TITLE
New site banner

### PR DESCRIPTION
**Pull request by iamdamion**
Added file and edited mkdocs.yml to add the banner that announces the new website.

### CODE REVIEW

**Comment by iamdamion** [comment]
@bendhouseart please review and merge.

File: overrides/main.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+      margin: 0
+    }
+  </style>
+  <div class="announcement">
+
+    <h2><b>THIS SITE IS BEING MERGED</b><br></h2>
+
+    <h3>
+      The BIDS group is currently finalizing a NEW main website that will merge our existing websites into one central, easy to navigate resource. <br>
+      The <b><a href="https://bids-website.readthedocs.io" target="_blank">New BIDS Wesbite</a> (https://bids-website.readthedocs.io)</b>
```
Comment (bendhouseart) [inline overrides/main.html:20]:
```suggestion
      The <b><a href="https://bids-website.readthedocs.io" target="_blank">New BIDS Website</a> (https://bids-website.readthedocs.io)</b>
```

**Comment by Remi-Gau** [comment]
That seemed to have failed: https://github.com/bids-standard/execution-spec/actions/runs/9571586107/job/26389022500#step:4:124

**Comment by bendhouseart** [comment]
> That seemed to have failed: https://github.com/bids-standard/execution-spec/actions/runs/9571586107/job/26389022500#step:4:124

Looks we don't actually check to see that the build passes/fails. @iamdamion I'll fix this (and hopefully it's either not an issue on the other repos or the same sop to make it work).

**Pull request merged**